src/views/join/join.jsx

```diff
@@ -6,11 +6,13 @@ const ErrorBoundary = require('../../components/errorboundary/errorboundary.jsx'
 const Page = require('../../components/page/www/page.jsx'); // eslint-disable-line no-unused-vars
 
 const openModal = true;
+const showCloseButton = false;
```

Comment:
i think it would be simpler here to just put these values in the render props below directly. They don't need to be top level-scoped variables.

Reply:
I like that better. Thanks for showing me how to do it :)